backend/models/Locations.ts

```diff
@@ -26,6 +28,7 @@ const schema = new mongoose.Schema(
       dislikes: { type: Array, default: [] }
     },
     filters: { type: Array, default: [] },
+    verificationStatus: { type: String },

```

Comment:
Maybe better use enum , like as role in the UserModel.ts scheme:
 role: {
      type: String,
      enum: ['user', 'admin', 'moderator'],
      default: 'user'
    },
?
 It could look like this: 
verificationStatus: {
      type: String,
      enum: ['verified', 'unverified', 'waiting'],
      default: 'unverified'
    },